WordPressKit/BlogServiceRemoteREST+Jetpack.swift

```diff
@@ -0,0 +1,55 @@
+public enum JetpackInstallError: String, Error {
+    case invalidCredentials = "INVALID_CREDENTIALS"
+    case forbidden = "FORBIDDEN"
+    case installFailure = "INSTALL_FAILURE"
+    case installResponseError = "INSTALL_RESPONSE_ERROR"
+    case loginFailure = "LOGIN_FAILURE"
+    case siteIsJetpack = "SITE_IS_JETPACK"
+    case activationOnInstallFailure = "ACTIVATION_ON_INSTALL_FAILURE"
+    case activationResponseError = "ACTIVATION_RESPONSE_ERROR"
+    case activationFailure = "ACTIVATION_FAILURE"
+    case unknown
```

Comment:
I saw the tests mention `UNKNOWN_ERROR`.  Should that be the string used here or was that just for convenience?

Reply:
Yes It's just for convenience. Only to test the default fallback to the `unknown` case.